setup.py

```diff
@@ -52,6 +52,7 @@ def version_scheme(version):
         "Programming Language :: Python :: 3.5",
         "Topic :: System :: Networking",
     ],
+    scripts=['sshuttle/bin/sudoers-add'],
```

Comment:
Possibly just `bin/sudoers-add` might be a better place here? Not sure this bash script needs to be installed in the Python directory too.

Reply:
I am not too familiar with how setup tools works and this was the way that worked for me.